templates/default/sql/dml/0.4.0.sql.erb

```diff
@@ -1,5 +1,6 @@
 DELETE FROM `hopsworks`.`variables` WHERE `id`='vagrant_enabled';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='rest_port';
 
-REPLACE INTO `hopsworks`.`variables` VALUES ("hopsexamples_version", "<%= node['hops']['hopsexamples_version'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("hopsexamples_version", "<%= node['hadoop_spark']['hopsexamples_version']
```

Comment:
@Limmen Why is this in `0.4.0.sql.erb` and not in `0.9.0.sql.erb`?

Reply:
@SirOibaf flyway will only run `0.9.0.sql`, it skips previous versions for dml files. Is that correct or am I missing something?

Reply:
@Limmen I had a chat with @SirOibaf and you are right, I had misunderstood how we parse these files since we changed it recently. Will merge.